float32    odom_x
float32    odom_y
float32    odom_z

float32    odom_lin_speed
int16       odom_ang_speed

int16    imu_yaw
int16    imu_pitch
int16    imu_roll

float32    bat_voltage
